chapters/ko/_toctree.yml

```diff
@@ -51,13 +51,16 @@
   sections:
   - local: chapter3/2
     title: 데이터 처리
-  - local: chapter3/5
-    title: 학습 곡선 이해하기
+
```

Comment:
It looks like there are two chapter 3 entries in the `toctree` and an un-added `chapter3/4` entry. The correct `toctree` should be the below I believe :)

```yaml
- title: 3. 🤗 사전훈련된 모델 미세조정
  sections:
  - local: chapter3/2
    title: 데이터 처리
  - local: chapter3/4
    title: 처음부터 구현하는 훈련 루프
  - local: chapter3/5
    title: 학습 곡선 이해하기
```